.github/workflows/test.patch.yml

```diff
@@ -46,6 +46,14 @@ jobs:
     steps:
       - run: 'echo "No build required"'
 
+  test-lightwalletd-integration:
+    name: Test integration with lightwalletd
+    runs-on: ubuntu-latest
+    needs: build
+    if: ${{ github.event.inputs.regenerate-disks != 'true' }}
```

Comment:
This condition will always be false.

In general, patch workflows need the opposite conditions to the original workflow. But in this case, we know the result of the condition will always be true, so we can just delete it.

```suggestion
```